jsontoxml.py

```diff
@@ -261,6 +242,31 @@ def getMessage(msg):
     return text
 
 
+def getParticipantInfo(participant_data):
+
+    name = participant_data['fallback_name']
+
+    if 'phone_number' in participant_data.keys():
+        phone = participant_data['phone_number']['e164']
+
+        if 'i18n_data' in participant_data['phone_number'].keys():
+            is_valid = participant_data['phone_number']['i18n_data']['is_valid']
+            if not is_valid:
+                # likely a short code, skip
+                return None
+        else:
+            return None
+    elif 'gaia_id' in participant_data['id'].keys():
+        id = participant_data['id']['gaia_id']
+        phone = input(f"Please provide the phone number (in E.164 format) for {name} (Google user ID {id}): ")
```

Comment:
So, I like this concept and I think that it might have value for a subset of users. However, I think that this might be better suited if it were triggered by a command line option. Maybe -c/--convert "optionally prompt for a phone number when a thread is not associated with one, ie. a non-SMS Hangouts conversation". I think there's a lot of value in allowing users to opt-in for this particular feature.

Reply:
Added an optional argument for prompting and updated the `README`.